package-dev/Plugins/iOS/SentryNativeBridge.m

```diff
@@ -1,6 +1,8 @@
-#import <Sentry/PrivateSentrySDKOnly.h>
-#import <Sentry/SentryOptions+HybridSDKs.h>
 #import <Sentry/Sentry.h>
+#import <MetricKit/MetricKit.h>
```

Comment:
I have to add `#import <MetricKit/MetricKit.h>`. Otherwise it will break in `Sentry-Switft.h` with
```
@interface SentryMXManager : NSObject <MXMetricManagerSubscriber> -- No type or protocol named 'MXMetricManagerSubscriber'
```